base/math.jl

```diff
@@ -360,6 +360,14 @@ asinh(x::Number)
     expm1(x)
 
 Accurately compute ``e^x-1``.
+# Examples
+```jldoctest
+julia> exp(1e-16) - 1 ≈ expm1(1e-16)
```

Comment:
Using `isapprox` here might be a bit confusing, because from a first look, it's not really clear that this is just because you didn't specify an absolute tolerance. I would probably just evaluate these two expressions in spearate lines, so it's clear what each of them returns.

Reply:
Also should we also add into the docstring that `function avoids the loss of precision involved in the direct evaluation of exp(x)-1 for small x` ?

Reply:
That sounds good to me!